src/GovUk.Education.ExploreEducationStatistics.Content.Model/Repository/MethodologyRepository.cs

```diff
@@ -44,7 +48,9 @@ public async Task<Methodology> CreateMethodologyForPublication(Guid publicationI
                             PublicationId = publicationId
                         }
                     }
-                }
+                },
+                Created = createdDate,
```

Comment:
Since the createdDate and user are always going to be set to the current time / current user, I'd be inclined to not pass them in as arguments and just set them here - not sure if there is some advantage to passing them in as arguments?

Reply:
It's just personal preference really - ever so slightly easier to test, and a bit more flexible in its use, that's all.  Happy either way